src/scripts/vscripts/pregame.lua

```diff
@@ -3962,6 +3966,20 @@ function Pregame:getActivePlayers()
     return total
 end
 
+-- Get host player
+function Pregame:getHostPlayer()
+	for i=0,24 do
```

Comment:
Aside from this, everything seems to be fine. Use GetPlayerCount instead and remove IsValidPlayer.